frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/summaryRelease.jspf

```diff
@@ -12,6 +12,7 @@
 <%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
 
 <core_rt:if test="${release != null}">
+<div class="up_Summary" id="up_Summary"><p><sw360:out value="${release.comment}" maxChar="500"/></p></div>
```

Comment:
Formatting:
```suggestion
    <div class="up_Summary" id="up_Summary"><p><sw360:out value="${release.comment}" maxChar="500"/></p></div>
```

And why is it limited to 500 chars ?

Reply:
@maxhbr The 500 chars limit is present in Component description as well. (file: `/sw360-portlet/src/main/webapp/html/components/includes/components/summary.jspf`).
So to make the release description ( view mode) in sync with component, I added that char limit.
Let me know if this has to be removed.